.pre-commit-config.yaml

```diff
@@ -2,6 +2,23 @@ ci:
   autoupdate_schedule: quarterly
 
 repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      #- id: check-docstring-first
+      - id: check-json
+      - id: check-yaml
+      - id: check-toml
+      #- id: check-case-conflict
```

Comment:
Do these checks (`docstring-first` and `case-conflict`) fail at the moment?

Reply:
no, they just were not part of my suggestion so was feeling hesitant to include them without your approval :rabbit: